src/components/SearchBox.vue

```diff
@@ -56,6 +56,10 @@ export default {
     ...mapActions('search', ['doSearch']),
     submitSearch: function () {
       this.doSearch(this.searchQuery)
+    },
+    submitMobileSearch: function () {
+      document.body.classList.toggle('menu-open')
```

Comment:
If you need to manipulate the DOM in this way in components then try to use this.$el instead. That will give you a few advantages and safe guards.